orchestra/static/orchestra/todos/todo-qa.directive.es6.js

```diff
@@ -14,8 +14,38 @@ export default function qa () {
     controller: function (todoApi, todoQaApi, $scope) {
       var todoQa = this
       todoQa.todos = []
+      todoQa.project = null
       todoQa.ready = false
 
+      todoQa.copyToClipboard = str => {
+        const el = document.createElement('textarea')
+        el.value = str
+        document.body.appendChild(el)
+        el.select()
+        document.execCommand('copy')
+        document.body.removeChild(el)
+      }
+
+      todoQa.commentSummary = (todos, summary) => {
+        summary = summary || ''
+        todos.forEach((todo) => {
+          summary = todoQa.commentSummary(todo.items, summary)
+          if (todo.qa && !todo.qa.approved && todo.qa.comment) {
+            summary = `*Todo*: ${todo.description}\n*Comment*: ${todo.qa.comment}\n\n${summary}`
```

Comment:
related: do you want to tell them you disapproved of certain QA items even if you didn't comment? no worries if you think that's a bad idea.

Reply:
It might be a bad idea since it wouldn't useful if we leave no comment. Especially since we want to use the summary to initiate a discussion around the feedback.

Reply:
Considering that motivation of summary is to initiate a discussion, adding certain QA items without comment will not help.